android/assets/jsons/translations/German.properties

```diff
@@ -10,6 +10,18 @@
 StartWithCapitalLetter = true
 
 
+# Fastlane
+# These will be automatically copied to the fastlane descriptions used by F-Droid. Their keys are not as usual the english original, please read those directly as linked.
+
+# Documentation: https://f-droid.org/en/docs/Build_Metadata_Reference/#Summary
+# English to translate: https://github.com/yairm210/Unciv/blob/master/fastlane/metadata/android/en-US/short_description.txt
+Fastlane_short_description = 4X-Spiel: Zivilisationen bauen
```

Comment:
Isn't this exactly what we try to avoid with these translation files? So that translators can easily see what needs to be translated and don't have to open another link (isn't always what you want when you only have a mobile device)?